spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +62,24 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with path to sqlite3 file' do
+      it 'returns a dataframe' do
+        result = Daru::IO::SqlDataSource.make_dataframe(db_name, query)
```

Comment:
It does work. I can write a test but it seems redundant. Should I move this test into the dataframe_spec.rb?